ui/app/styles/core/helpers.scss

```diff
@@ -171,7 +171,7 @@
   margin-bottom: $spacing-s;
 }
 .has-bottom-margin-m {
-  margin-bottom: $spacing-m;
+  margin-bottom: $spacing-m !important;
```

Comment:
Can we do the same here?

Reply:
It should be removed in the latest files. I committed it here. https://github.com/hashicorp/vault/pull/14250/commits/85f9d139994f7002c3ed64de7d375c99322e146f

Reply:
Ah thank you!